leap/leap.rb

```diff
@@ -0,0 +1,10 @@
+class Year
+  def self.leap?(year)
+    if(year%4==0&&year%100!=0||year%400==0)
+  	  return true
+    else
+  	  return false
+    end
```

Comment:
no need to return true and false like this. condition it self return true and false.